src/services/scheduler.ts

```diff
@@ -203,30 +204,52 @@ export async function init() {
   }
 }
 
-// Primitive RPC between parent and child process. Requires the parent to produce a unique ID per
-// request to pair up to the correct response. May replace with a revived multitransport-jsonrpc?
-process.on('message', (m: string[]) => {
-  const [fn, ...args] = m;
-  switch (fn) {
-    case 'init':
-      init()
-        .then(() => process.send?.(['initComplete', ...args, undefined]))
-        .catch(e => process.send?.(['initComplete', ...args, e.message]));
-      break;
-    case 'start':
-      start(args[0], args[1])
-        .then(() => process.send?.(['startComplete', ...args, undefined]))
-        .catch(e => process.send?.(['startComplete', ...args, e.message]));
-      break;
-    case 'stop':
-      stop(args[0])
-        .then(() => process.send?.(['stopComplete', ...args, undefined]))
-        .catch(e => process.send?.(['stopComplete', ...args, e.message]));
-      break;
-    case 'stopAll':
-      stopAll()
-        .then(() => process.send?.(['stopAllComplete', ...args, undefined]))
-        .catch(e => process.send?.(['stopAllComplete', ...args, e.message]));
-      break;
+if (require.main === module) {
+  const app = express();
+  const port = 14527;
+
+  function respondErrorAndDie(res: any, err: any) {
+    res.status(500).send(err);
+    logger.error(`Scheduler exited with error: ${err}`);
```

Comment:
we might want to port over the sentry logic so this, and other exceptions, get logged to sentry. that can happen in a follow up PR though

Reply:
Do we have a ticket for it? I can take it next if we do. If not, can you brief me on it so that I can go for it?

Reply:
Yes, basically we want exceptions within the scheduler to also go to sentry. Currently, only the routes in the `db.ts` router get this treatment from the main Express server: https://github.com/iasql/iasql-engine/blob/main/src/index.ts. Basically we want to port the sentry setup logic from that file to this one. You can test that it works locally but setting the sentry logic in the config here:

> Some considerations regarding local run of iasql-engine after the merge:
> 
> * When running iasql-engine on your local computer, you'll need to start an instance of `dist/services/scheduler.js` along with the main process. If you don't you'll get an error regarding the engine is unable to connect to the scheduler.
>   
>   * Same logic happening in the `docker-entrypoint.sh`

Starting the docker container locally would spin it up for us, right?